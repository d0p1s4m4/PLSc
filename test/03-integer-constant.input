123 0xFFF 0t4984frpko 0b010101 0bFF 