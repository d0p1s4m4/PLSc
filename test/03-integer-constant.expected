{
  "file": "test/03-integer-constant.input",
  "tokens": [
    {
      "type": "NUMBER",
      "value": "123",
      "position": {
        "column": 1,
        "line": 1,
        "length": 3
      }
    },
    {
      "type": "NUMBER",
      "value": "0xfff",
      "position": {
        "column": 5,
        "line": 1,
        "length": 5
      }
    },
    {
      "type": "NUMBER",
      "value": "0b010101",
      "position": {
        "column": 23,
        "line": 1,
        "length": 8
      }
    }
  ],
  "errors": [
    {
      "type": 3,
      "position": {
        "column": 11,
        "line": 1,
        "length": 11
      }
    },
    {
      "type": 3,
      "position": {
        "column": 32,
        "line": 1,
        "length": 4
      }
    }
  ]
}
